Repository/AbstractPhpcrRepository.php

```diff
@@ -110,4 +112,12 @@ protected function unresolvePath($path)
      * @return ArrayResourceCollection
      */
     abstract protected function buildCollection(array $nodes);
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getVersions($path)
+    {
+        return new VersionList($path, [new CmfResource($path)]);
```

Comment:
We do this:

https://github.com/puli/repository/blob/1.0/src/AbstractRepository.php#L35

On Thu, May 19, 2016 at 08:28:45AM -0700, Maximilian Berghoff wrote:
>    In [1]Repository/AbstractPhpcrRepository.php:
> 
>  > @@ -110,4 +112,12 @@ protected function unresolvePath($path)
>  >       * @return ArrayResourceCollection
>  >       */
>  >      abstract protected function buildCollection(array $nodes);
>  > +
>  > +    /**
>  > +     * {@inheritdoc}
>  > +     */
>  > +    public function getVersions($path)
>  > +    {
>  > +        return new VersionList($path, [new CmfResource($path)]);
> 
>    [2]@dantleech you are sure that is just a dummy output for us, or is there
>    some reason? Should we put $repository->get(path) into the list?
> 
>    —
>    You are receiving this because you were mentioned.
>    Reply to this email directly or [3]view it on GitHub
> 
>    Reverse link: [4]unknown
> 
> References
> 
>    Visible links
>    1. https://github.com/symfony-cmf/resource/pull/15#discussion_r63900135
>    2. https://github.com/dantleech
>    3. https://github.com/symfony-cmf/resource/pull/15/files/318868bc8aae59467545426bf044226df2519641#r63900135
>    4. https://github.com/symfony-cmf/resource/pull/15/files/318868bc8aae59467545426bf044226df2519641#r63900135